docs/tutorials/analysis/1D/spectral_analysis_rad_max.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>In the context:</p><ul><li>Link to the GADF when you refer to it</li><li>Minor: <code>Makers</code> not <code>Maker</code>s </li></ul><p>In the Introduction:</p><ul><li>Normally the <code>ReflectedRegionsFinder</code> is never introduced, it works under the hood. Maybe not refer to it here as well, and simply say that the important part of this analysis is to explicitly specify a <code>WobbleRegionsFinder</code> which supports OFF regions with sizes varying with energy, which is directly taken from the predefined RadMax values.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3712/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>All done!</p>

Reply:
<p>Removed "the latter"</p>